Forwarded-Upstream: not needed
Author: Evgeni Golov <evgeni@debian.org>
Comment:
 Disable model tests that require an internet connection.
 .
 This is a Debian specific patch.

Index: python-django-1.2.3/tests/modeltests/validation/tests.py
===================================================================
--- python-django-1.2.3.orig/tests/modeltests/validation/tests.py	2010-10-23 14:59:12.447389830 +0200
+++ python-django-1.2.3/tests/modeltests/validation/tests.py	2010-10-23 15:01:01.875390381 +0200
@@ -52,14 +52,6 @@
         mtv = ModelToValidate(number=10, name='Some Name', url='not a url')
         self.assertFieldFailsValidationWithMessage(mtv.full_clean, 'url', [u'Enter a valid value.'])
 
-    def test_correct_url_but_nonexisting_gives_404(self):
-        mtv = ModelToValidate(number=10, name='Some Name', url='http://google.com/we-love-microsoft.html')
-        self.assertFieldFailsValidationWithMessage(mtv.full_clean, 'url', [u'This URL appears to be a broken link.'])
-
-    def test_correct_url_value_passes(self):
-        mtv = ModelToValidate(number=10, name='Some Name', url='http://www.djangoproject.com/')
-        self.assertEqual(None, mtv.full_clean()) # This will fail if there's no Internet connection
-
     def test_text_greater_that_charfields_max_length_eaises_erros(self):
         mtv = ModelToValidate(number=10, name='Some Name'*100)
         self.assertFailsValidation(mtv.full_clean, ['name',])
